File: JS/UberX.js
class UberX extends Car {
    constructor(license, driver, brand, model){
        super(license, driver)
        this.brand = brand;
        this.model = model;
    }

    printDataCar(){
        super.printDataCar();
        console.log(`Model: ${this._model} Brand: ${this._brand}`);
    }

}

File: C#/UberX.cs
namespace UberPoo
{

    class UberX : Car
    {
        public string brand;
        public string model;

        public UberX(string license, Account driver, string brand, string model) : base(license, driver)
        {
            this.brand = brand;
            this.model = model;
        }

        public new void PrintDataCar(){
            base.PrintDataCar();
            System.Console.WriteLine("Model: "+ model + " Brand: " + brand);
        }
    }
}